src/main/java/dev/j3fftw/litexpansion/Events.java

```diff
@@ -131,8 +132,11 @@ public void onObsidiaDrill(PlayerInteractEvent e) {
         final Location blockLocation = block.getLocation();
 
         final DiamondDrill diamondDrill = (DiamondDrill) SlimefunItem.getByID(Items.DIAMOND_DRILL.getItemId());
-        if (blockType.equals(Material.OBSIDIAN) && diamondDrill.isItem(e.getItem())
-            && SlimefunPlugin.getProtectionManager().hasPermission(e.getPlayer(), blockLocation, ProtectableAction.BREAK_BLOCK)
+
+        if ((diamondDrill.isItem(e.getItem())
+            && drillableBlocks.contains(blockType) || blockType.equals(Material.OBSIDIAN))
+            && SlimefunPlugin.getProtectionManager().hasPermission(e.getPlayer(),
+            blockLocation, ProtectableAction.BREAK_BLOCK)
```

Comment:
should also break ores
and should be enchanted

Reply:
If it breaks ores, it will drop the ore block, unless I add special cases for every ore, or at this point I can change it to a pickaxe and fix all that

Reply:
its okay for it dropping the ore block